src/app/stop-watch/stop-watch.component.ts

```diff
@@ -0,0 +1,57 @@
+import { Component, OnInit } from '@angular/core';
+import { Subject, takeUntil, debounceTime, buffer, filter, map, BehaviorSubject, withLatestFrom, switchMap, timer, take, skip } from "rxjs";
+
+@Component({
+    selector: 'app-stop-watch',
+    templateUrl: './stop-watch.component.html',
+    styleUrls: ['./stop-watch.component.scss'],
+})
+export class StopWatchComponent implements OnInit {
+    time$ = new BehaviorSubject<number>(0);
+
+    onStart$ = new BehaviorSubject<boolean>(false);
+    waitTrigger$ = new Subject<void>();
+
+    ngOnInit(): void {
+        const clickStream = this.waitTrigger$.asObservable();
+
+        clickStream.pipe(
+            buffer(clickStream.pipe(debounceTime(300))),
+            map(list => list.length),
+            filter((clicksQuantity) => clicksQuantity >= 2),
```

Comment:
why >=, we want it to work for 3 clicks as well?

Reply:
done ✅